policyengine_us/parameters/gov/states/oh/tax/income/credits/earned_income_credit_rate.yaml

```
@@ -0,0 +1,11 @@
+description: Ohio matched the federal earned income tax credit at this rate.
+values:
+  2021-01-01: 0.3
```

Since `Effective:July 3, 2019`
```suggestion
  2020-01-01: 0.3
```